src/decision/include/GpsDecision.h

```diff
@@ -14,15 +14,51 @@
 #include <geometry_msgs/Twist.h>
 #include <geometry_msgs/Point.h>
 #include <ros/ros.h>
+#include <std_msgs/Float32.h>
 
 class GpsDecision {
 public:
     GpsDecision(int argc, char **argv, std::string node_name);
-private:
+
+    /**caclutes the distance between wayPoint and current point
+     * @ parameter: given the gps locaiton of the next move
+     * @ return: the distance between the next location and the current location
+     **/
+    static double distance(const geometry_msgs::Point::ConstPtr& relative_gps);
+
+    /**
+     * @ parameter
+     * relative_gps: given the gps locaiton of the next move
+     * current_heading: the curren heading relative to north in degrees (0 to 360 degrees)
+     * @ return:
+     *   desiredAngle: the angle(less than PI) in degrees that robot need to turn
+     *   positive value means turning clockwise
+     *   negative value means turing counter-clockwise
+     */
+    static double desiredAngle(const geometry_msgs::Point relative_gps,float current_heading,geometry_msgs::Point currentPoint);
+    /**
+     * @ parameter
+     *  desiredAngle: the angle(less than PI) in degrees that robot need to turn
+     *                 positive value means turning clockwise
+     *                 negative value means turing counter-clockwise
+     *  angular_velocity: the angular_velocity(rad/s)
+     * @ effects: rotate the robot by giving angles and angular velocity
```

Comment:
is `@effects` a valid flag? I've never seen it before. (It might be, I'm just asking to make sure)